pom.xml

```diff
@@ -69,27 +70,159 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<jdkLevel>1.8</jdkLevel>
-		<slf4j.version>1.7.32</slf4j.version>
+		<slf4j.version>1.7.35</slf4j.version>
 		<junit.version>4.13.1</junit.version>
+		<mockito-core.version>3.12.4</mockito-core.version>
+		<hamcrest-all.version>1.3</hamcrest-all.version>
+		<junit.jupiter.version>5.6.1</junit.jupiter.version>
```

Comment:
If we have Jupiter, do we still need JUnit 4.13.1?

Reply:
I'll check that.

Reply:
Jupiter is used in the code and dictionary generator tests but I recall JUnit compatibility problems with the maven plugin testing harness so the repository-to-qfj maven plugin module uses Junit 4.  Also there are many other places in the overall project that use junit4 which have not been changed - in this PR.
For ease of migration to junit5 there is a Junit Vintage runtime that requires Junit 4.12 or later.
"JUnit 5 = JUnit Platform + JUnit Jupiter + JUnit Vintage"
https://junit.org/junit5/docs/current/user-guide/
So I think it's good to have JUnit in the dependency management for now.

<img width="335" alt="image" src="https://user-images.githubusercontent.com/29655222/180422554-38ad2561-c819-41dd-8f0a-42e08601a609.png">